v10: Update articles with references to embedded resources (#4077)
umbraco/UmbracoDocs: Extending/Backoffice-Tours/index.md

---
versionFrom: 9.0.0

> versionTo: 10.0.0

Reviewer: I guess this needs to be removed again if we update this article to make it work for 10.
As I guess the backoffice tours are not meant to leave?

Author: I will merge this PR in now as is.

I have a task on my board, on running some more tests with these tours - in relation to the tests we ran at our previous meeting.

- **Core tours and custom tours**

>     The tour that ship with Umbraco are embedded into the CMS.
> 
>     When you want to implement custom tours into your website, we recommend that you create

Reviewer 2: ```suggestion
    The tour that ship with Umbraco are embedded into the CMS assemblies.
```

It's not supported. People needs to do like for plugins
